src/components/TodoItem.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
```

Comment:
Kindly avoid disabling the linters because linters make your code clean and follow standards.

Reply:
There seems to be a problem with my Code linters checker, so i put that there to help the program run well.